src/containers/pages/InterventionPlan/NewPlan/test/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Router } from 'react-router';
+import NewPlan from '../index';
+import { someObject } from './fixtures';
+
+const history = createBrowserHistory();
+
+describe('containers/pages/NewPlan', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders without crashing', () => {
+    shallow(<NewPlan />);
+  });
+
+  it('renders correctly', () => {
+    const wrapper = mount(
+      <Router history={history}>
+        <NewPlan />
+      </Router>
+    );
+
+    // check that page title is displayed
+    expect(toJson(wrapper.find('h3.mb-3.page-title'))).toMatchSnapshot('page title');
+
+    // check that planform receives the correct props
+    expect(JSON.stringify(wrapper.find('PlanForm').props())).toEqual(JSON.stringify(someObject));
```

Comment:
1. Why do we have to call `JSON.stringify` on both sides?
2. Can we name it something better than `someObject`?